oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -488,6 +488,14 @@ public void close() {
 
     private void runWhenPermitted() {
         if (indexStats.isPaused()) {
+            if (indexStats.forcedLeaseRelease){
+                try {
+                    clearLease();
+                } catch (CommitFailedException e) {
+                    log.warn("Unable to release lease, please try again");
```

Comment:
Please log the exception (message and stack trace):

    log.warn("Unable to release lease, please try again", e);